docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -5,6 +5,26 @@
 :frontmatter-tags-content-type: [how-to] 
 :frontmatter-tags-user-goals: [analyze]
 
+To change case closure options and add custom fields, templates, and connectors for external incident management systems, go to *Cases* -> *Settings*.
+
+[role="screenshot"]
+image::images/cases-settings.png[Shows the case settings page]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+NOTE: To view and change case settings, you must have the appropriate {kib} feature privileges. Refer to <<case-permissions>>.
+
+[float]
+[[close-sent-cases]]
+=== Case closures
+
+If you close cases in your external incident management system, the cases will remain open in {elastic-sec} until you close them manually.
+
+To close cases when they are sent to an external system, select *Automatically close cases when pushing new incident to external system*.
+
+[float]
+[[cases-ui-integrations]]
```

Comment:
@lcawl this anchor name is replicated in the topic anchor [here](https://github.com/elastic/security-docs/pull/5619/files#diff-8b48d6fdee2c77e4e55132f57ed70591eda137487ace3f73b843eb6229b3fb57L1) -- or maybe it's the other way around? How did you want to handle this, and do we need to fix this in any other branches?